open_academy/models/session.py

```diff
@@ -32,29 +32,29 @@ def _onchange_invalid_values(self):
         if self.number_of_seats < 0:
             return {
                 'warning': {
-                    'title': "Something bad happened.",
-                    'message': "Number of seats must be greater than 0."
+                    'title': ("Something bad happened."),
```

Comment:
where is the _??

Reply:
Without the `_` the terms are not translated.

<img width="987" alt="Screen Shot 2022-09-08 at 4 23 17 p m" src="https://user-images.githubusercontent.com/47955967/189228419-2c131f78-dfb0-4671-b950-2c54bf59954a.png">

Reply:
to reproduce, remove DB, install your module and then install the language and trigger the error

Reply:
Sorry, when I added de new .po file for translations they all appear in that file so I thought that the translations will be made automatically with the .po file, I will correct that.

Reply:
I am getting this error when I add the translations 
![image](https://user-images.githubusercontent.com/76703666/189234218-3c609d17-2fdc-487a-99ae-8bde0bc81594.png)

Reply:
You need to import it